TypeCobol.LanguageServer/Completion Factory/CompletionFactory.cs

```diff
@@ -233,7 +233,7 @@ public static IEnumerable<CompletionItem> GetCompletionForProcedureParameter(Pos
                         completionItems.ForEach(ci => ci.insertText += " OUTPUT ");
                 if (lastSignificantToken.TokenType == TokenType.IN_OUT && alreadyGivenParametersCount == (procedureSignatureContext.Profile.InoutParameters.Count - 1))
                         completionItems.ForEach(ci => ci.insertText += " OUTPUT ");
-                if (lastSignificantToken.TokenType == TokenType.OUTPUT && alreadyGivenParametersCount == (procedureSignatureContext.Profile.OutputParameters.Count - 1))
+                if (lastSignificantToken.TokenType == TokenType.OUTPUT && alreadyGivenParametersCount == (procedureSignatureContext.Profile.OutputParameters.Count - 1) && !codeElement.ConsumedTokens.Last().InputStream.ToString().EndsWith("."))
```

Comment:
You should better use the type of the ConsumedToken. A "." is of type PeriodSeparator